Cargo.toml

```diff
@@ -20,7 +20,7 @@ derive-new = "0.5"
 rmpv = "0.4.4"
 rust-embed = { version = "5.2.0", features = ["debug-embed"] }
 image = "0.22.3"
-nvim-rs = { git = "https://github.com/kethku/nvim-rs", features = [ "use_tokio" ] }
+nvim-rs = { git = "https://github.com/KillTheMule/nvim-rs", branch = "serial", features = [ "use_tokio" ] }
```

Comment:
Get this merged upstream first. It's bad practice to change the dependencies to another remote for maintenance reasons and security reasons.

Reply:
Just realized that you are upstream. lol

Reply:
I'll test it up in a bit